benchmark/check_until.yaml

```
@@ -0,0 +1,15 @@
+prelude: |-
+  $LOAD_PATH.unshift(File.expand_path("lib"))
+  require "strscan"
+  scanner = StringScanner.new("test string")
+  str = "string"
+  reg = /string/
+benchmark:
+  regexp_literal: |
```

```suggestion
  regexp: |
```

Or adding `_literal` to the `string` label.

---

OK.
I see.